.github/workflows/check_code_format.yml

```diff
@@ -22,14 +22,17 @@ jobs:
 
       - name: Format code
         run: |
-          git clean -f -x -d
+          git clean -f -x -d -e ".nim_runtime"
           nim prettyfy
 
       - name: Fail if needs reformatting
         run: |
+          echo ".nim_runtime" > .gitignore
           if [[ $(git status --porcelain) ]]; then
              echo "please reformat/prettyfy these files:"
              git status --porcelain=v1
+             rm -vf .gitignore
              exit 1
           fi
+          rm -vf .gitignore
```

Comment:
Since you add `.nim_runtime` to `.gitignore`, are all of these changes needed?

Reply:
Sorry but I will avoid the .gitignore file finally, let me push a last change.